coordinator/src/node/storage.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use diesel::r2d2::ConnectionManager;
+use diesel::r2d2::Pool;
+use diesel::PgConnection;
+use lightning::chain::keysinterface::SpendableOutputDescriptor;
+use lightning::chain::transaction::OutPoint;
+use lightning::ln::PaymentHash;
+use lightning::ln::PaymentPreimage;
+use lightning::ln::PaymentSecret;
+use ln_dlc_node::node;
+use ln_dlc_node::HTLCStatus;
+use ln_dlc_node::MillisatAmount;
+use ln_dlc_node::PaymentFlow;
+use ln_dlc_node::PaymentInfo;
+
+#[derive(Clone)]
+pub struct NodeStorage {
+    pool: Pool<ConnectionManager<PgConnection>>,
+}
+
+impl node::Storage for NodeStorage {
+    // Payments
+
+    fn insert_payment(&self, _payment_hash: PaymentHash, _info: PaymentInfo) -> Result<()> {
+        todo!()
+    }
+    fn merge_payment(
+        &self,
+        _payment_hash: &PaymentHash,
+        _flow: PaymentFlow,
+        _amt_msat: MillisatAmount,
+        _htlc_status: HTLCStatus,
+        _preimage: Option<PaymentPreimage>,
+        _secret: Option<PaymentSecret>,
+    ) -> Result<()> {
+        todo!()
+    }
+    fn get_payment(
+        &self,
+        _payment_hash: &PaymentHash,
+    ) -> Result<Option<(PaymentHash, PaymentInfo)>> {
+        todo!()
+    }
+    fn all_payments(&self) -> Result<Vec<(PaymentHash, PaymentInfo)>> {
+        todo!()
+    }
```

Comment:
This will all be implemented as soon as we merge @da-kami's work on https://github.com/get10101/10101/issues/936.